tests/util/usedPropTypes.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../../lib/rules/no-unused-prop-types');
+
+const parsers = require('../helpers/parsers');
+
+const parserOptions = {
+  ecmaVersion: 2018,
+  sourceType: 'module',
+  ecmaFeatures: {
+    jsx: true
+  }
+};
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({parserOptions});
+
+ruleTester.run('no-unused-prop-types', rule, {
+  valid: [
+    {
```

Comment:
This valid code fails without the fix
![image](https://user-images.githubusercontent.com/748586/74104168-64470980-4b52-11ea-81d1-40ed6f8e7bef.png)